docs/AWSIoTSiteWiseSource.md

```diff
@@ -206,3 +211,28 @@ Type: Boolean
 
 Type: Boolean
 
+---
+
+## SiteWiseDataSourceSettings
+
+Optional settings that can be provided when initializing the AWS IoT SiteWise source.
+
+```
+import { initialize } from '@iot-app-kit/source-iotsitewise';
+
+const { IoTSiteWiseClient } = require("@aws-sdk/client-iotsitewise");
+
+const iotsitewiseClient = new IoTSiteWiseClient({ region: "REGION" });
+
+const { query } = initialize({ iotsitewiseClient, settings: { batchDuration: 100 } });
+```
+
+`batchDuration`
```

Comment:
the preceding paragraph should be re-arranged a little. The very first word should be (Optional) [basic description]